server/server.py

```diff
@@ -2,23 +2,50 @@
 import os
 import json
 from tornado.web import RequestHandler, Application
+import numpy as np
+import pandas as pd
 import tensorflow as tf
 from smart_news_query_embeddings.preprocessing.bert_tokenizer import \
 create_tokenizer, tokenize_data
 
 home = os.path.expanduser('~')
 bert_dir = os.path.join(home, 'smart-news-query-embeddings/uncased_L-12_H-768_A-12/')
 tokenizer = create_tokenizer(bert_dir)
-exp_dir = os.path.join(home, 'smart-news-query-embeddings/experiments', 'baseline_model', 'model')
+exp_name = 'baseline_model'
+exp_dir = os.path.join(home, 'smart-news-query-embeddings/experiments', exp_name, 'model')
 keras_model = tf.keras.models.load_model(exp_dir)
+embeddings_path = os.path.join(home, 'smart-news-query-embeddings/experiments', exp_name, 'embeddings')
```

Comment:
Add some empty lines inside this large block, add some one-liner comments to explain the blocks